modules/archetypes/lib/policy_assignments/policy_assignment_es_deploy_mdfc_config.tmpl.json

```diff
@@ -1,66 +1,82 @@
 {
-  "name": "Deploy-MDFC-Config",
   "type": "Microsoft.Authorization/policyAssignments",
-  "apiVersion": "2019-09-01",
+  "apiVersion": "2022-06-01",
+  "name": "Deploy-MDFC-Config",
+  "location": "${default_location}",
+  "dependsOn": [],
+  "identity": {
+    "type": "SystemAssigned"
+  },
   "properties": {
     "description": "Deploy Microsoft Defender for Cloud and Security Contacts",
     "displayName": "Deploy Microsoft Defender for Cloud configuration",
-    "notScopes": [],
+    "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policySetDefinitions/Deploy-MDFC-Config",
+    "enforcementMode": null,
+    "nonComplianceMessages": [
+      {
+        "message": "Microsoft Defender for Cloud and Security Contacts {enforcementMode} be deployed."
+      }
+    ],
     "parameters": {
       "emailSecurityContact": {
-        "value": "security_contact@replace_me"
+        "value": "defaultString4"
       },
       "logAnalytics": {
-        "value": "${root_scope_id}-la"
+        "value": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/${root_scope_id}-mgmt/providers/Microsoft.OperationalInsights/workspaces/${root_scope_id}-la"
```

Comment:
Fix this value

Reply:
Fixed

Reply:
@matt-FFFFFF and @jtracey93. This does appear to work, however the policy is failing because the `-asc-export` resource group does not exist. Should that resource group have been created? Also, do you think the value of this property should have been this in the first place as opposed to just the workspace name like it was previously?

Reply:
![image](https://user-images.githubusercontent.com/1612200/235304054-825f12ad-d73d-46e9-8f07-7c8a3eaea64e.png)

Reply:
![image](https://user-images.githubusercontent.com/1612200/235304131-c6e30f33-7443-4c17-95da-0552144b1df3.png)